playbooks/roles/integreatly/defaults/main.yml

```diff
@@ -152,4 +152,4 @@ integreatly_osd_upgrade_name: "Integreatly_Upgrade_[OSD]"
 integreatly_osd_upgrade_workflow_name: "Integreatly_Bootstrap_and_Upgrade_[OSD]"
 integreatly_osd_upgrade_desc: "Workflow to upgrade integreatly on an OSD cluster"
 integreatly_osd_update_inventory: OSD_update_inventory
-integreatly_osd_update_branch: "release-1.4.0"
\ No newline at end of file
+integreatly_osd_update_branch: "release-1.4.0"
```

Comment:
Looks like we have other references for OSD tasks. This should read in the integreatly_version IMO so that it would be set to:
```
integreatly_osd_update_branch: "{{ integreatly_version }}"
```

I'll fix this in a separate PR